server/search/search.controller.ts

```diff
@@ -125,7 +135,12 @@ export class SearchController {
                     req.language,
                     null
                 ),
-                this.sentenceService.findAll(searchText, parsedPageSize, null),
+                this.sentenceService.findAll(
+                    searchText,
+                    parsedPageSize,
+                    null,
```

Comment:
suggestion: make it optional in the function statement and just omit the parameter when calling the function